src/services/users/reset-password/reset-password.test.js

```diff
@@ -0,0 +1,24 @@
+const chai = require('chai')
+const verifyResetToken = require('./verifyPasswordResetToken')
+const generateAccessToken = require('../generate-access-token')
+
+const testEmail = 'testemail@gmail.com'
+
+        
+const { expect } = chai
+
+describe('verifyPasswordResetToken', () => {
```

Comment:
the test cases in the file will be more appropriate for `verify-password-reset-token.test.js`